### DISCUSSION TITLE
Admin UI roles

### DISCUSSION THREAD

**Discussion by thal-sh** [pocketbase/pocketbase#3827] [Q&A] [answered]
I really like pocketbase, simple and easy to use but when I'm developing for clients they tend to need an admin UI, when are we expecting admin dashboard roles?

**Comment by ganigeorgiev** [comment, marked as answer]
Please check the discussions in https://github.com/pocketbase/pocketbase/discussions/3784, https://github.com/pocketbase/pocketbase/discussions/94#discussioncomment-3725219, https://github.com/pocketbase/pocketbase/discussions/50.

In short - there are no ETAs and for now this remains out of the roadmap but eventually may reconsider it after we manage to finalize most of the v1 tasks.
